pkg/verifier/cosign/cosign.go

```diff
@@ -119,7 +120,16 @@ var logOpt = logger.Option{
 // used for mocking purposes
 var getKeysMaps = getKeysMapsDefault
 
-const verifierType string = "cosign"
+const (
+	verifierType         string = "cosign"
```

Comment:
Correct. They come from the cosign library. In particular, the Cosign CLI uses it. When you do a `cosign verify` it prints out some verification "headers" which are a human readable list of all the verification performed. I think it's pretty nice to have. In fact something that notation might want to consider too.

Reply:
could we add comment link to src of these strings in  cosign code base if possible? if they gets updated, do you think we need to update our strings too?

Reply:
Unfortunately, cosign library doesn't store strings as constants so we cannot use them directly. Yes, we'd likely need to update strings on our side if they were updated on cosign library in the future.

Reply:
As discussed in PR meeting, I will 1. add header with url to where in cosign library messages are from and 2. add TODO to each line so future editors can cross-reference cosign library implementation to see if the messages have been updated since there's no good way to track those changes on Ratify side